features/networking/egress-ip.feature

```diff
@@ -354,11 +354,12 @@ Feature: Egress IP related features
     """
 
     # Create a pod
-    Given I obtain test data file "routing/caddy-docker.json"
+    Given I obtain test data file "routing/web-server-rc.yaml"
```

Comment:
`web-server-rc.yaml` creates pod and svc as well, seems here you don't need the service, if that's true then just use `web-server-1.yaml`.

Reply:
correct.  updated, thanks